function_builder.go

```diff
@@ -0,0 +1,38 @@
+package function
+
+import (
+	"fmt"
+)
+
+const (
+	BuilderPack    = "pack"
+	BuilderS2i     = "s2i"
+	DefaultBuilder = BuilderPack
+)
+
+func AllBuilders() []string {
+	return []string{BuilderPack, BuilderS2i}
+}
+
+func ValidateBuilder(builder string, allowUnset bool) (errors []string) {
```

Comment:
It looks like this function only ever returns a single error. Why have the signature indicate an array?